packages/progress-bar/src/ProgressBar.stories.tsx

```diff
@@ -150,6 +158,23 @@ export const WithHeaderTruncation: StoryObj<typeof ProgressBar> = {
   },
 };
 
+export const WithProviderDarkMode: StoryObj<typeof ProgressBar> = {
+  parameters: {
+    backgrounds: {
+      default: 'dark',
+      values: [
+        { name: 'dark', value: color.dark.background.secondary.default },
+      ],
+    },
+  },
+  render: _args => (
```

Comment:
don't even need to declare args `() => `

Reply:
actually we should probably pass in these args. Props like `variant` etc. don't work in Storybook (we should either make them work, or remove them)